massa-network-worker/src/binders.rs

```diff
@@ -34,11 +31,9 @@ impl WriteBinder {
     ///
     /// # Argument
     /// * msg: date to transmit.
-    pub async fn send(&mut self, msg: &Message) -> Result<u64, NetworkError> {
+    pub async fn send(&mut self, buf: &[u8]) -> Result<u64, NetworkError> {
```

Comment:
Why don't you use the Message struct anymore ?

Reply:
So that in the case of a block and header, we can reuse the serialized form stored in storage(without having to do that inside the binder).